portal/src/main/scala/cromwell/pipeline/ApplicationComponents.scala

```diff
@@ -11,7 +11,8 @@ import cromwell.pipeline.utils.ApplicationConfig
 import scala.concurrent.ExecutionContext
 
 final class ApplicationComponents(
-  implicit val config: Config = ConfigFactory.load(),
+  implicit
+  val config: Config = ConfigFactory.load(),
```

Comment:
Is it ok? "Implicit" and "val" on different lines?

Reply:
seems like we will have to leave, cuz there are no too much options to format implicits in scalafmt yet